Translations update from Fedora Weblate

weblate opened the pull request:
Translations update from [Fedora Weblate](https://translate.fedoraproject.org) for [fedora-infra/noggin](https://translate.fedoraproject.org/projects/fedora-infra/noggin/).



Current translation status:

![Weblate translation status](https://translate.fedoraproject.org/widgets/fedora-infra/-/noggin/horizontal-auto.svg)

abompard commented:
Hi translation team! There are a couple commits missing the `Signed-off-by` line, could you fix that please?

jconstanti committed:
Translated using Weblate (Catalan)

Currently translated at 1.9% (3 of 151 strings)

Co-authored-by: josep constantí <iceberg.jcm@gmail.com>
Signed-off-by: josep constantí <iceberg.jcm@gmail.com>
Translate-URL: https://translate.fedoraproject.org/projects/fedora-infra/noggin/ca/
Translation: fedora-infra/noggin

TaoMonLae committed:
Added translation using Weblate (Burmese)

Co-authored-by: Tao Mon Lae <tamonlae@gmail.com>

Atalanttore committed:
Translated using Weblate (German)

Currently translated at 86.0% (130 of 151 strings)

Co-authored-by: Ettore Atalan <atalanttore@googlemail.com>
Signed-off-by: Ettore Atalan <atalanttore@googlemail.com>
Translate-URL: https://translate.fedoraproject.org/projects/fedora-infra/noggin/de/
Translation: fedora-infra/noggin

bittin committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (296 of 296 strings)

Translated using Weblate (Swedish)

Currently translated at 100.0% (151 of 151 strings)

Added translation using Weblate (Swedish)

Co-authored-by: Luna Jernberg <bittin@reimu.nl>
Signed-off-by: Luna Jernberg <bittin@reimu.nl>
Translate-URL: https://translate.fedoraproject.org/projects/fedora-infra/noggin/sv/
Translation: fedora-infra/noggin

atriwidada committed:
Added translation using Weblate (Indonesian)

Co-authored-by: Andika Triwidada <andika@gmail.com>

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://translate.fedoraproject.org/projects/fedora-infra/noggin/
Translation: fedora-infra/noggin

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 100.0% (296 of 296 strings)

Translated using Weblate (Korean)

Currently translated at 100.0% (296 of 296 strings)

Translated using Weblate (Korean)

Currently translated at 44.9% (133 of 296 strings)

Co-authored-by: 김인수 <simmon@nplob.com>
Signed-off-by: 김인수 <simmon@nplob.com>
Translate-URL: https://translate.fedoraproject.org/projects/fedora-infra/noggin/ko/
Translation: fedora-infra/noggin

copperii committed:
Translated using Weblate (Finnish)

Currently translated at 66.5% (197 of 296 strings)

Co-authored-by: Jan Kuparinen <copper_fin@hotmail.com>
Signed-off-by: Jan Kuparinen <copper_fin@hotmail.com>
Translate-URL: https://translate.fedoraproject.org/projects/fedora-infra/noggin/fi/
Translation: fedora-infra/noggin
